web/components/Feed/Feed.tsx

```diff
@@ -45,10 +46,14 @@ export type Option = {
   label: string;
 };
 
-export type SelectedFilters<T extends Filter> = {
-  [key in T["key"]]: Option[];
+export type SelectedFilters = {
+  [key: string]: Option[];
```

Comment:
Is there a way you can make this type more specific? The keys are limited to the filter keys